buildSrc/src/main/kotlin/common.kt

```diff
@@ -27,19 +28,22 @@ fun Project.common() {
   tasks.withType<KotlinCompile>()
     .configureEach {
 
-      kotlinOptions {
-        allWarningsAsErrors = false
-
-        jvmTarget = "1.8"
-
-        freeCompilerArgs = freeCompilerArgs + listOf(
-          "-Xjvm-default=all",
-          "-Xallow-result-return-type",
-          "-opt-in=kotlin.contracts.ExperimentalContracts",
-          "-opt-in=kotlin.Experimental",
-          "-opt-in=kotlin.time.ExperimentalTime",
-          "-opt-in=kotlin.RequiresOptIn",
-          "-Xinline-classes"
+      compilerOptions {
+        allWarningsAsErrors.set(false)
+
+        jvmTarget.set(JVM_1_8)
+
+        freeCompilerArgs.set(
+          freeCompilerArgs.get() + listOf(
+            "-Xjvm-default=all",
+            "-Xallow-result-return-type",
+            "-opt-in=kotlin.contracts.ExperimentalContracts",
+            "-opt-in=kotlin.Experimental",
+            "-opt-in=kotlin.time.ExperimentalTime",
+            "-opt-in=kotlin.RequiresOptIn",
+            "-opt-in=org.jetbrains.kotlin.compiler.plugin.ExperimentalCompilerApi",
```

Comment:
After upgrading to Kotlin 1.8 I was getting a ton of `ExperimentalCompilerApi` so I added an opt in here.